app/assets/stylesheets/incidents.scss

```diff
@@ -71,3 +71,8 @@ table.history>tbody>tr{
     }
   }
 }
+
+.high-contrast-options option{
+  color: black;
+  background: white;
```

Comment:
Yeah, it's a bit of a crap-shoot. Styling form elements has always been terrible which is why so many UI frameworks would just replace them with lists and divs and such.

Interestingly, on *my* version of Firefox, **setting** a background drastically changes how the element looks:

![screenshot_2017-06-12_12-59-51](https://user-images.githubusercontent.com/240275/27045619-6fce00b0-4f6f-11e7-974c-55bf557140df.png)
![screenshot_2017-06-12_13-01-14](https://user-images.githubusercontent.com/240275/27045618-6fcd70aa-4f6f-11e7-8dad-075b2b94bc3f.png)

Dunno.